Exploración del dataset. 

In [12]:
import pandas as pd

df = pd.read_csv('tot_reg_desap.csv') # lectura del dataset

print(f"Filas: {df.shape[0]}")
print(f"Columnas: {df.shape[1]}")

print(df.columns.to_list()) #ver las categorias de las columnas

# Periodo de fechas (1. ordenamos respecto a la fecha de desaparición, 2. Seleccionamos la primera y última)
df_ord_fechas = df.sort_values(by='fecha_desap').dropna() #quitamos los NaN

primero_y_ultimo = df_ord_fechas.iloc[[0, -1]]
print("\nPrimera Fecha y última fecha:")
print(primero_y_ultimo['fecha_desap'])

# Imprimir head para analizar el significado de las entradas del dataset
df.head()

Filas: 89720
Columnas: 13
['Unnamed: 0', 'nombrecompleto', 'rnpdno_json', 'entidad_desap', 'pre_rnped', 'rnped', 'rnpdno_csv', 'bg_1', 'estatus_busqueda', 'bg_2', 'fub', 'DC_id', 'fecha_desap']

Primera Fecha y última fecha:
82335    01/01/1967
10573    31/12/2022
Name: fecha_desap, dtype: str


,Unnamed: 0,nombrecompleto,rnpdno_json,entidad_desap,pre_rnped,rnped,rnpdno_csv,bg_1,estatus_busqueda,bg_2,fub,DC_id,fecha_desap
0,1,LUIS ADRIAN JUAREZ MORALES,1,QUERETARO,0,0,1,0,NaN,0,NaN,"101 (json), 5673 (csv)",19/04/2023
1,2,ERNESTO ALONSO HERNANDEZ GUTIERREZ,1,QUERETARO,0,0,1,1,Con indicios / Ubicada,1,187731B1B-3302-4BE3-81D5-018CFB86A00B,"349 (json), 448 (csv), 187731B1B-3302-4BE3-81D...",12/07/2016
2,3,PIEDAD JESUS GUERRERO CHAVEZ,1,QUERETARO,0,0,1,1,Se busca reportante,1,122009919-199C-4B90-8462-935A6EDCE439,"732 (json), 14618 (csv), 122009919-199C-4B90-8...",26/09/2017
3,4,LORENZO MARTINEZ,1,QUERETARO,0,0,1,1,Se requieren datos de identidad,1,148436E49-364B-4663-847C-0338E23B2460,"1007 (json), 89640 (csv), 148436E49-364B-4663-...",17/03/2010
4,5,AGUSTINA OLVERA OCHOA,1,QUERETARO,0,0,1,1,NaN,0,NaN,"1013 (json), 57259 (csv)",04/06/2023


En base a lo anterior podemos observar que:

1. El dataset tiene 89720 filas y 13 columnas
2. El periodo de fechas cubierto está en el rango entre 1 de enero de 1967 hasta el 31 de diciembre del 2022 *pero creo que hay un error con los métodos con los que obtuve esa información porque posteriormente en el head me salen registros del 2023*
3. Cada entrada en el dataset corresponde a la información de una persona desaparecida. Entre lo más importante se encuentra el nombre, la entidad donde desaperecio, el estado de búsqueda y la fecha de desaparición.